src/cloud-element-templates/util/bindingTypes.js

```diff
@@ -11,7 +11,8 @@ export const MESSAGE_PROPERTY_TYPE = 'bpmn:Message#property';
 export const MESSAGE_ZEEBE_SUBSCRIPTION_PROPERTY_TYPE = 'bpmn:Message#zeebe:subscription#property';
 export const ZEEBE_CALLED_ELEMENT = 'zeebe:calledElement';
 export const ZEEBE_LINKED_RESOURCE_PROPERTY = 'zeebe:linkedResource';
-export const ZEEBE_USER_TASK = 'zeebe:userTask';
+export const ZEEBE_USER_TASK = 'zeebe:userTask'; // TODO: I thing this should also be a Binding Type
```

Comment:
@yT0n1 todo follow up separately

Reply:
@barmac do you have an idea why userTask is not included in `EXTENSION_BINDING_TYPES`? Moreover, it seems like the array is used in a function called `unsetProperty` which does not seem to be used anymore at all .